internal/service/dns_config/api_view_resource_test.go

```diff
@@ -0,0 +1,1955 @@
+}
+
+func TestAccViewResource_EcsZones(t *testing.T) {
+	var resourceName = "bloxone_dns_view.test_ecs_zones"
+	var name = acctest.RandomNameWithPrefix("view")
+	var v dns_config.ConfigView
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccViewEcsZones(name, "allow", "tf-infoblox.com."),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckViewExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "ecs_zones.0.access", "allow"),
+					resource.TestCheckResourceAttr(resourceName, "ecs_zones.0.fqdn", "tf-infoblox.com."),
+				),
+			},
+			// Update and Read
+			{
+				Config: testAccViewEcsZones(name, "deny", "tf-test-infoblox.com."),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckViewExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "ecs_zones.0.access", "deny"),
+					resource.TestCheckResourceAttr(resourceName, "ecs_zones.0.fqdn", "tf-test-infoblox.com."),
+				),
+			},
+			// Delete testing automatically occurs in TestCase
+		},
+	})
+}
+
+func TestAccViewResource_EdnsUdpSize(t *testing.T) {
+	var resourceName = "bloxone_dns_view.test_edns_udp_size"
+	var name = acctest.RandomNameWithPrefix("view")
+	var v dns_config.ConfigView
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccViewEdnsUdpSize(name, 1232),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckViewExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "edns_udp_size", "1232"),
+				),
+			},
+			// Update and Read
+			{
+				Config: testAccViewEdnsUdpSize(name, 1000),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckViewExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "edns_udp_size", "1000"),
+				),
+			},
+			// Delete testing automatically occurs in TestCase
+		},
+	})
+}
+
+func TestAccViewResource_FilterAaaaAcl(t *testing.T) {
+	var resourceName = "bloxone_dns_view.test_filter_aaaa_acl"
+	var name = acctest.RandomNameWithPrefix("view")
+	var v dns_config.ConfigView
+
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { acctest.PreCheck(t) },
+		ProtoV6ProviderFactories: acctest.ProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read
+			{
+				Config: testAccViewAclIP("filter_aaaa_acl", name, "allow", "192.168.10.10"),
+				Check: resource.ComposeTestCheckFunc(
+					testAccCheckViewExists(context.Background(), resourceName, &v),
+					resource.TestCheckResourceAttr(resourceName, "filter_aaaa_acl.0.access", "allow"),
```

Comment:
you should also check for filter_aaaa_acl.0.element. Same comment applies to the other ACL tests

Reply:
Resolved